source/terms.txt

```diff
@@ -0,0 +1,169 @@
+
+   * - **type**
+     - | **Description:**
+       | "A type determines a set of values together with operations and methods specific to those values" (`Types <https://golang.org/ref/spec#Types>`__)
+       | **Recommendation:**
+       | Use this when describing a data type, including categorical (pointer, function, strcut, array, string, etc.). The term "type" should not receive any special formatting, but a specific type name should be monospaced.
+
+   * - **pointer**
+     - | **Description:**
+       | References to values/records. See `Go by Example: Pointers <https://gobyexample.com/pointers>`__ for further explanation and examples.
+       | **Recommendation:**
+       | We can talk about pointers as ``*C`` or "pointer to ``C``". The latter may be easier for a new developer to understand, and to discover the meaning through a google search.
+
+   * - **struct, struct type**
+     - | **Description:**
+       | "A type determines a set of values together with operations and methods specific to those values" (`Struct types <https://golang.org/ref/spec#Struct_types>`__)
+       | An example of a struct:
+
+       .. code-block:: go
+
+          type color struct {
+            name string
+            r,g,b int
+          }
+
+       | **Recommendation:**
+       | Use struct or struct type to refer to both the declaration within a package as well as the data held by a variable.
+       | Example(s):
+
+       - "The color struct type contains three ``int`` fields."
+       - ``c := color(name: "crimson", r: 153, g: 0, b: 0)`` Call ``isPrimary() on the ``color`` struct to determine whether it is a primary color.
+
+   * - **variable**
+     - | **Description:**
+       | Holds data whether a data structure or pointer type and is accessed by the name by which it was declared. See `Variables <https://golang.org/ref/spec#Variables>`__ for more info. The following shows a few example variable declarations:
+
+       .. code-block:: go
+
+          var c = color.NewColor("chartreuse")
+          x := color.NewColor("indigo")
+          _, g, _ := c.RGBValues()
+          var pColor *Color
+
+       | **Recommendation:**
+       | Use "variable" when you are referencing one but do not require any specificity. If it is more helpful to refer to the variable by its name, use that instead.
+       | Example:
+
+       - "You can store the return value in a value so you can access it later."
+
+   * - **pointer reciever**
+     - | **Description:**
+       | "You can declare methods with pointer receivers. This means the receiver type has the literal syntax ``*T`` for some type ``T``. (Also, ``T`` cannot itself be a pointer such as ``*int``.)" (from `Pointer receivers <https://tour.golang.org/methods/4>`__.
+       | **Recommendation:**
+       | We decided as a team that no further specificity is needed when using this term in prose, but can use the ``*`` prefix when it is featured in a list of links to API documentation.
+
```

Comment:
I was surprised a definition for object was not included. Are we avoiding the term "object" in our go documentation? I support not using "object", but I thought you were in favor of using object (as it is technically correct). 

I am only writing this comment, as this was surprising given how I understood your stance on the term "object" in golang.